packages/ts-common/src/report/interfaces/report.interface.ts

```diff
@@ -0,0 +1,36 @@
+import { AuditInterface, ReferenceIdInterface } from '@concepta/ts-core';
+import { ReportStatusEnum } from '../enum/report-status.enum';
+// import { FileOwnableInterface } from '../../file/interfaces/file-ownable.interface';
+
+/**
+ * Interface representing a report entity
+ */
+export interface ReportInterface extends ReferenceIdInterface, AuditInterface {
+  //  FileOwnableInterface
```

Comment:
erroneous comment

Reply:
i thought about using the ownable pattern but i realize we are not using on rockets